src/material/timepicker/timepicker.md

```diff
@@ -1 +1,310 @@
+will report the `matTimepickerParse` error. The string is parsed using the `parseTime` method of
+the [the current date implementation](#choosing-a-date-implementation-and-format-settings).
+
+The timepicker input also checks that the value typed in by the user fits within the minimum and
+maximum bounds set through the `matTimepickerMin` and `matTimepickerMax` inputs. They accept either
+a date object with a specific time or a time string. The inputs also control which times will be
+shown inside of the dropdown menu. For example, setting `matTimepickerMin="12:30"` and
+`matTimepickerMax="21:25"` will allow the user to only select a time between 12:30 in the afternoon
+and 9:25 in the evening. If the value is outside of those bounds, either a `maxTimepickerMin` or
+`matTimepickerMax` error will be reported to the value accessor.
+
+<!-- example(timepicker-validation) -->
+
+
+### Customizing the dropdown options
+
+By default the `mat-timepicker` dropdown shows options at 30 minute intervals. You can customize the
+list of options either by setting an interval or providing a custom set of options.
+
+The easiest way is to change the options is to pass the `interval` input to `mat-timepicker` with
+an interval string which will be used when generating the options. For example,
+`<mat-timepicker interval="90m"/>` will show the options at 90 minute intervals, starting from the
+minimum time and ending at the maximum. Valid interval strings include:
+* A number which will be interpreted as minutes, e.g. `interval="50"` represents 50 minutes.
+* A number with short units, for example `30m` represents 30 minutes while `5h` is 5 hours.
+Supported short units include `h`/`H` for hours, `m`/`M` for minutes and `s`/`S` for seconds.
+* A number with long units, for example `75 min` represents 75 minutes while `1.5 hours` is an hour
+and a half. Supported long units include `min`/`minute`/`minutes` for minutes, `hour`/`hours` for
+hours and `second`/`seconds` for seconds.
+
+Furthermore, the default interval can be controlled for the entire application using the
+`MAT_TIMEPICKER_CONFIG` injection token. For example, adding the following to your `providers` will
+default all timepickers to a 90 minute interval:
+
+```ts
+import {MAT_TIMEPICKER_CONFIG} from '@angular/material/timepicker';
+
+{
+  provide: MAT_TIMEPICKER_CONFIG,
+  useValue: {interval: '90 minutes'},
+}
+```
+
+If your app requires more fine-grained control over the options, you can pass in an array of
+options into `mat-timepicker` instead. Note that the options need to match the `MatTimepickerOption`
+interface.
+
+<!-- example(timepicker-options) -->
+
+
+### Customizing the toggle icon
+
+`mat-timepicker-toggle` renders a clock icon by default. You can customize it by projecting in an
+element with the `matTimepickerToggleIcon` attribute into the toggle:
+
+<!-- example(timepicker-custom-icon) -->
+
+### Internationalization
```

Comment:
It's probably good to have an example for this. Datepicker has an example for this too: https://material.angular.io/components/datepicker/overview#datepicker-locale

Reply:
Added.